Tests/js/views/services/plugins/assets/ez-searchplugin-tests.js

```diff
@@ -3,10 +3,25 @@
  * For full copyright and license information view LICENSE file distributed with this source code.
  */
 YUI.add('ez-searchplugin-tests', function (Y) {
-    var locationSearchTests, locationSearchEventTests,
+    var locationSearchTests, locationSearchEventTests, _configureQueryAndContentServiceMock,
         contentSearchTests, loadResourcesTests, registerTest,
         Assert = Y.Assert, Mock = Y.Mock;
 
+    _configureQueryAndContentServiceMock = function (context, queryMethodName) {
+        Mock.expect(context.query, {
+            method: queryMethodName,
+            args: [Mock.Value.Object],
+        });
+
+        Mock.expect(context.contentService, {
+            method: 'createView',
+            args: [context.query, Mock.Value.Function],
+            run: Y.bind(function () {
+                Y.Mock.verify(context.query);
```

Comment:
you should not do that here, you have no guarantee this will be called. You should do that at the end of each test.